CRM/Mailing/BAO/Mailing.php

```diff
@@ -28,7 +28,7 @@
 /**
  *
  * @package CRM
- * @copyright CiviCRM LLC (c) 2004-2018
+ * @copyright CiviCRM LLC (c) 2004-2017
```

Comment:
Can we remove the header changes?  They should be changed en-masse at some point and it may cause merge conflicts with this unrelated PR.